### PR TITLE
Remove Reporter + Add browser/os detection + No longer import library everytime

### PR DESCRIPTION
**Pull request by EWhite613**
Fixes: 
* https://github.com/ciena-blueplanet/ember-cli-visual-acceptance/issues/2

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -40,57 +57,60 @@ function getImage(req, res) {
...
     }))
     app.use(bodyParser.json())
-    app.get('/image', function(req, res) {
+    app.get('/image', function (req, res) {
       getImage(req, res)
     })
 
-    app.post('/image', function(req, res) {
+    app.post('/image', function (req, res) {
+      console.log('Hello')
```
Comment (sglanzer) [inline index.js]: Missed console log

File: tests/integration/simple-test.js [hunk 2]
```diff
@@ -13,15 +13,15 @@ describeComponent(
...
     integration: true
   },
-  function() {
-    it('renders', function(done) {
+  function () {
+    it('renders', function (done) {
       this.render(hbs `{{input type="text" value='Simple words' disabled=entryNotAllowed size="50"}}`)
       expect(this.$()).to.have.length(1)
-      visualAcceptance('Simple', null, null, 0.00).then(function(data) {
+      visualAcceptance('Simple', null, null, 0.00).then(function (data) {
```
Comment (sglanzer) [inline tests/integration/simple-test.js]: Can we rename `visualAcceptance` to `capture` if it's quick?

**Comment by sglanzer** [comment]
Approved - looks good.  Would you mind adding the README updates as well?

**Comment by EWhite613** [comment]
@sglanzer Whoops forgot branches were a thing. So also in this PR is the fix for container resizing since html2canvas just scales the image. And adding the container resolution to the image path

**Comment by sglanzer** [comment]
Approved with the new changes, nice work!

**Pull request merged**